docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md

```diff
@@ -0,0 +1,78 @@
+# Manually Installing Windows Prerequisites for OpenEnclave
+
+The following are prerequisites for building and running Open Enclave on
+Windows. These are required for both SGX1 mode and SGX1+FLC mode
+
+- Intel® X86-64bit architecture with SGX1 or SGX2
+- A version of Windows OS with native support for SGX features:
+   - For server: Windows Server 2016 (or newer)
+   - For client: Windows 10 64-bit with Fall Creators Update (1709) or newer
+- [Microsoft Visual Studio 2017](https://visualstudio.microsoft.com/vs/older-downloads/)
```

Comment:
Were you planning to remove this and the section on Visual Studio 2017 below in favor of the x64 Native Tools download as reflected in the Getting Started instructions? If not, there's a weird disconnect here.

Reply:
@CodeMonkeyLeet I am wondering how someone would install just X64 native tools. I am thinking as part of the VS2017/2019 install, you can choose only the x64/x86 build tools. Is there a separate native tools install package?

Reply:
I see that @jazzybluesea has later commented on using the Native tools installer. Will follow up